### DISCUSSION TITLE
Problem with FFD block visualization after optimization

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#456] [Bug]
Hi 

In the latest versions of DAFoam 3.0.6 or 3.0.2 the command  (“writeDeformedFFDs”: True) does not work anymore. Is there any way to visualize FFD block after deformation/optimization?

Thank you in advance for your reply.

**Comment by friedenhe** [comment]
Yeah, we lost the writeDeformedFFDs feature... The main issue is that this feature seems to be broken in the new pyGeo version...

**Reply by hosseinnegahban** [reply]
Is there any alternative way to use? I urgently need this feature to be included in my paper. I would appreciate it if you help me

**Comment by friedenhe** [comment]
You have to manually deform the FFD and save them using a stand-alone pyGeo run script.. Check an example of how to use the `writeTecplot` function from [here]( https://github.com/mdolab/pygeo/blob/86e8e9b2e15ba166b1218327eb34698087bfecbb/examples/ffd_cylinder/runFFDExample.py#L114)

**Reply by raisePiginYourhome** [reply]
how to export the .xyz mesh file from ICEM CFD? I try to export .dat format and it show the garbled code.

**Reply by friedenhe** [reply]
Convert to multi-block structured mesh and then export to plot3d format. Check ICEM's manual.

**Reply by raisePiginYourhome** [reply]
![pic2](https://github.com/mdolab/dafoam/assets/126868299/517ac483-5c6e-4fb0-87a4-f0a0ebb584ab)
which option I should choose?

**Reply by friedenhe** [reply]
precision - double. Include IBLANK array - no. Use the default for all other options.

**Reply by raisePiginYourhome** [reply]
Thannk you, I'll try this.
